util/getUniqueCurrentProjects.test.js

```diff
@@ -0,0 +1,48 @@
+import getUniqueCurrentProjects from './getUniqueCurrentProjects';
+
+describe('getUniqueCurrentProjects', () => {
+  const MOCK_TEAM_MEMBERS_DATA = {
+    '0001': {
+      currentProjects: [
+        {
+          client: 'Thorin Oakenshield',
+          project: 'Quest of Erebor',
+          startDate: '2022-04-27',
+          endDate: '2022-11-23',
+        },
+      ],
+    },
+    '0002': {
+      currentProjects: [
+        {
+          client: 'Thorin Oakenshield',
+          project: 'Quest of Erebor',
+          startDate: '2022-04-27',
+          endDate: '2022-11-23',
+        },
+        {
+          client: 'White Council',
+          project: 'Dol Guldur',
+          startDate: '2022-05-02',
+          endDate: '2022-09-10',
+        },
+      ],
+    },
+    '0003': {
+      currentProjects: [
+        {
+          client: 'Internal',
+          project: 'Lonely Mountain Residency',
+          startDate: '1851-04-12',
+          endDate: '2022-06-03',
+        },
+      ],
+    },
+  };
+
+  it('returns unique current projects from all team members as an array in alphabetical order', () => {
+    const currentProjects = getUniqueCurrentProjects(MOCK_TEAM_MEMBERS_DATA);
+
+    expect(currentProjects).toEqual(['Dol Guldur', 'Lonely Mountain Residency', 'Quest of Erebor']);
+  });
+});
```

Comment:
I think it would be smart to add some tests for some edge cases. Some thoughts, what if a currentProjects is empty `currentProjects : []`, what if the a project doesn't have a name just a client. What if a there are some null or undefined values floating around in the data. We want to make sure this code still behaves as expected in these cases.

Reply:
🏆 Good looking out!